app/controllers/forms_controller.rb

```diff
@@ -58,7 +69,7 @@ def update
       @form.transaction do
         # @form.updated_by = current_user
         @form.form_questions.destroy_all
-        @form.form_questions = create_form_questions(params[:form][:linked_questions], params[:form][:linked_response_sets])
+        @form.form_questions = create_form_questions
         # When we assign update_successful, it is the last expression in the block
         # That means, if the form fails to update, this block will return false,
         # which will cause the transaction to rollback.
```

Comment:
Comments aren't counted, but I managed to get this fixed in other ways. I think we could probably increase the class size limit to 150 or 200 in rubocop, since 100 is pretty punishing, but I didn't need to do so in this case.